bpf/bpf_lxc.c

```diff
@@ -11,6 +11,8 @@
 
 #include <linux/icmpv6.h>
 
+#define IS_BPF_LXC 1
+
 /* Controls the inclusion of the CILIUM_CALL_SRV6 section in the object file.
  */
 #define SKIP_SRV6_HANDLING
```

Comment:
Why not use `#ifndef SKIP_NODEPORT_NAT` instead of `declare_tailcall_if`? As you can see above, we already use the first convention today.

Reply:
Hi, I saw both conventions being used, but as I wanted to conditionally only declare tail calls, I decided it'd be more natural to use `declare_tailcall_if` instead of putting the whole code under macro.